libs/generators/src/lib/generators/generators-menu/generators-menu.component.spec.ts

```diff
@@ -1,14 +1,22 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { Router } from '@angular/router';
 
 import { GeneratorsMenuComponent } from './generators-menu.component';
 
 describe('GeneratorsMenuComponent', () => {
   let component: GeneratorsMenuComponent;
   let fixture: ComponentFixture<GeneratorsMenuComponent>;
+  let routerMock: any;
```

Comment:
Im not sure about the type here, should I use partial<Router> ??

Reply:
Why not use the RouterTestingModule?

Reply:
@OmerGronich Thanks, forgot it was a thing.
anyway its not relevant anymore because Ive changed to routerLink.